src/client.js

```diff
@@ -118,6 +119,15 @@ class Client {
   }
 
   /**
+   * A wrapper around the constructor, for backwards compatability.
```

Comment:
so, what's the point of having both methods? why have `.buildClient` versus passing in a `Config` object to `ShopifyBuy`?